src/components/styles/Layout.tsx

```diff
@@ -27,3 +28,7 @@ export const PageSider = styled(Sider)`
   max-width: ${siderWidth} !important;
   min-width: ${siderWidth} !important;
 `;
+
+export const SimplePageTitle = styled(H2)`
+  margin-bottom: 16px;
+`;
```

Comment:
these seem like one-off styled per-page that should not exist as shared components

Reply:
This title component can be used for any page title that follows the style of the My Patches page (a page title without a badge). I can rename this component so we can find it easier.